packages/website/components/enterpriseTierRequestModal/enterpriseTierRequestModal.js

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useLayoutEffect } from 'react';
+import kwesforms from 'kwesforms';
+
+import Modal from 'modules/zero/components/modal/modal';
+import CloseIcon from 'assets/icons/close';
+import Button from 'components/button/button.js';
+import GradientBackground from 'components/gradientbackground/gradientbackground';
+
+const UserRequestModal = ({ isOpen, onClose }) => {
+  useEffect(() => {
+    kwesforms.init();
+  }, []);
+
+  useLayoutEffect(() => {
+    const form = document.getElementById('enterpriseKwesForm');
+    form?.addEventListener('kwSubmitted', function () {
+      // do we need to do any custom logic?
```

Comment:
🤷  I don't think so? @drewdelano what did you have in mind?

If nothing in particular, will you remove this?

I think because it does a DOM query by element id, if there are several of these on the same page, it's not guaranteed that it will select the HTMLElement that corresponds to this component instance. (there may also be react-specific concerns too as I think react reserves the right to destroy/recreate those HTML elements whenver it wants which would remove the listener).

because of all that I try to just avoid using `id` attributes altogether in reusable components.

Reply:
Yeah, I think you're right.  I copied it from the older kwes-form usages.